Translation-Polish.md

```diff
@@ -0,0 +1,499 @@
+
+Spójrzmy na to:
+
+```tex
+% main.tex
+\documentclass[a4paper]{article}
+
+\begin{document}
+
+Hello Latex, This is my first part.
+
+Hello Latex, This is my second part.
+
+\end{document}
+```
+
+Tak zapisywaliśmy to do tej pory. Teraz podzielmy to na pliki i zaimportujmy je za pomocą `\input`:
+
+```tex
+% main.tex
+\documentclass[a4paper]{article}
+
+\begin{document}
+
+Hello Latex, This is my first part.
+
+\input{second_file}
+
+\end{document}
+```
+
+```tex
+% second_file.tex
+Hello Latex, This is my second part.
+```
+
+Główny plik wygląda teraz inaczej, ale jest z pewnością lepiej zorganizowany.
+
+![](https://camo.githubusercontent.com/e4d46edfad527c4c9be54cf8f1725c2fc52039d6/68747470733a2f2f7331342e706f7374696d672e6f72672f646567306b716875392f6d756c74695f66696c652e706e67)
+
+:bangbang: **Wskazówka**: Dla czytelności i świetego spokoju edytujacego zaleca się, by podzielić Twój plik systematycznie i metodycznie.
+
+## Przydatne narzędzia
+
+#### Dystrybucje
+
+- [MiKTeX](https://miktex.org/about) for Windows.
+- [TeX Live](https://www.tug.org/texlive/) for Linux and Unix-based.
+- [MacTeX](https://tug.org/mactex/) for macOS.
+- [ShareLaTeX](https://www.sharelatex.com/) — edytor online.
+- [Overleaf](https://www.overleaf.com/) — edytor online.
+- [StackEdit](https://stackedit.io/) — In-browser markdown editor.
+
+#### Edytory
+
+- [TeXMaker](http://www.xm1math.net/texmaker/) Cross platform LaTeX editor.
+- [TeXStudio](http://www.texstudio.org/) An enhanced fork of TeXMaker with more features.
+- TeXShop and TeXworks (minimal editors)
+
+#### Kompilatory
+
+- Większość edytorów pozwoli Ci na wybór domyślnego kompilatora:
+
+![](http://i.imgur.com/FbNUiL7.png)
+
+## HOORAY!!
+
+:tada: Dziękuję Ci za ukończenie tego poradnika. To wszystko czego potrzebujesz do rozpoczęcia swojej przygody z LaTeX. :hammer:  
+Jeśli będziesz potrzebował pomocy, [tutaj](http://www.latex-project.org/help/documentation/) znajdziesz wiele przydatnych informacji .
+
+## Licencja
+
+[![](http://www.wtfpl.net/wp-content/uploads/2012/12/wtfpl-badge-1.png)](http://www.wtfpl.net/)
+
+**DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE**
+Copyright (C) 2016 Luong Vo
+Everyone is permitted to copy and distribute verbatim or modified copies of this license document, and changing it is allowed as long as the name is changed.
+TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION AND MODIFICATION: You just DO WHAT THE FUCK YOU WANT TO.
+
+<div id='donation'/>
+
+[![](https://www.paypalobjects.com/en_US/i/btn/btn_donate_SM.gif)](https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=5ZG5Z47L2ZGYC)
+
+A beer in your country can buy a meal in mine.
+
+## Adnotacja od autora tłumaczenia
+
+Autorem tłumaczenia jest Artur Komorowski (aokomorowski@gmail.com). Jeśli widzisz w tekście błędy - proszę, popraw je. Jeśli uznasz, że LaTeX zmienił Twoje życie - podziękuj autorowi.
```

Comment:
I wouldn't recommend leaving your email in public. If you want to, you can leave a link to your GitHub profile.
```suggestion
Autorem tłumaczenia jest Artur Komorowski (@aokomorowski). Jeśli widzisz w tekście błędy - proszę, popraw je. Jeśli uznasz, że LaTeX zmienił Twoje życie - podziękuj autorowi.
```